test/dns_dnscmd/dnscmd_test.rb

```diff
@@ -3,61 +3,70 @@
 require 'dns_dnscmd/dns_dnscmd_main'
 
 class DnsCmdTest < Test::Unit::TestCase
+  def setup
+    Proxy::Dns::Plugin.load_test_settings(
+        :dns_server => 'a_server',
+        :dns_zones => ['bar.domain.local', 'domain.local', '33.168.192.in-addr.arpa', '168.192.in-addr.arpa'])
+  end
+
   def test_dnscmd_provider_initialization
```

Comment:
The test is still usefull -- it verifies that dnscmd respects parameters defined in the configuration file. However, plugin initialization code is redundant, could you remove it?

I see that the dns_zones setting is defined in the dnscmd config file, but is retrieved from the main plugin settings in the code. Test setup method is wrong too.